Rakefile

```diff
@@ -7,7 +7,6 @@ require File.expand_path('../config/application', __FILE__)
 Plots2::Application.load_tasks
 
 Rake::Task['test:run'].clear
-Rake::Task['test:all'].clear
```

Comment:
So does this PR no longer run the `test:all` commands on lines 20+ of this file? That includes Jasmine javascript tests and some formatting checks via Coveralls report. Or are those being run some other way?

Reply:
No, actually the thing is `rake test:all` was not performing the rails test well. As it is showing the same warning many many times and stops after encountering one error or failure. So, I guess it would be better if we run `rails test -d` and run the jasmine tests separately. What do you think?

Reply:
Yes let's try to get the Jasmine and coveralls tasks running however you think best. Thanks!